cli/src/android/update.ts

```diff
@@ -54,9 +54,10 @@ export async function installGradlePlugins(config: Config, capacitorPlugins: Plu
 
   const settingsPath = join(config.app.rootDir, 'android');
   const dependencyPath = join(config.app.rootDir, 'android', 'app');
+  const relativeCapcitorAndroidPath = relative(settingsPath, capacitorAndroidPath).replace('\\', '/');
```

Comment:
This only replaces the first occurrence, try with something like `.replace(/\\/g, '/');`